readme.md

```diff
@@ -883,7 +883,7 @@
 - [Reddit](https://www.reddit.com/r/node)
 - [Twitter](https://twitter.com/nodejs)
 
-### Miscellaneous
+### Various
```

Comment:
Makes no sense for the rule to enforce that on headings in different sections.

I've disabled the rule for now. (Next release).

Just revert this change.